test/common/testlib.py

```diff
@@ -746,8 +746,8 @@ def become_superuser(self, user=None, password=None):
         self.switch_to_top()
 
         self.open_superuser_dialog()
-        self.wait_in_text(".pf-c-modal-box:contains('Switch to administrative access')", f"Password for {user or 'admin'}:")
-        self.set_input_text(".pf-c-modal-box:contains('Switch to administrative access') input", password or "foobar")
+        self.wait_in_text(".pf-c-modal-box:contains('Switch to administrative access')", f"Password for {user or 'admin'}")
+        self.set_input_text(".pf-c-modal-box:contains('Switch to administrative access') input[type=password]", password or "foobar")
```

Comment:
Adding the `[type=password]` looks like a rebase error, this shouldn't be in the "Drop trailing : from sudo prompts commit (as most other places with this change are in the next one). So if you have to re-push at all for any reason, it'd be nice to clean this up. Otherwise, not important enough.

Reply:
> Adding the [type=password] looks like a rebase error, 

Yes, fixed.